tests/ClientTest.php

```diff
@@ -2528,9 +2516,7 @@ public function testOurSerializer($after_creation)
                     ],
                 ]
             );
-            $events = $client->getSentEvents();
-            $event = array_pop($events);
-            $this->assertEquals(Dummy_Serializer::$default_value, $event['extra']['foo']);
+            $serializer->__phpunit_verify();
```

Comment:
No we don't. we just have to test that the `Serializer::serialize` method is called when appropriate, e.g. when sending a message to the Sentry server. There is no reason to test that (this is just an example) a `boolean` is passed to the method